src/main/java/hudson/remoting/Launcher.java

```diff
@@ -701,9 +701,9 @@ private static void ttyCheck() {
             // we seem to be running from interactive console. issue a warning.
             // but since this diagnosis could be wrong, go on and do what we normally do anyway. Don't exit.
             System.out.println(
-                    "WARNING: Are you running slave agent from an interactive console?\n" +
+                    "WARNING: Are you running agent.jar from an interactive console?\n" +
```

Comment:
Would be better to avoid ".jar" here, Remotig lib may have another name && may be packaged into WAR/*.jnlp/whatever